### ISSUE TITLE
@Finished event?

### ISSUE THREAD

**Issue by xlcrr** [nandi95/vue-toastify#8]
Hi again,

Getting great use out of this plugin, thanks so much!

Would it be possible to add some kind of event listener that would call when the popup has disappeared or the user clicks it to remove it?

This would be useful for refreshing the screen, or loading data, after an event has completed successfully and the user has been notified of a successful event.

**Comment by nandi95** [comment]
At this moment (v1.*) I can only imagine achieving this by passing a function in the status object.
It only makes sense to execute the provided function on notification dismiss and finish. Not on the prompt mode and loader mode as you're already aware of them closing.
It would look something like this:

```
this.$vToastify.error({
    body: "I'm going to refresh the page"
    callback: () => {
        window.location.reload()
    }
);
```

**Comment by nandi95** [comment]
This has been addressed in version 1.1.0
@xlcrr
